### PR TITLE
German translation: alias, bash, borg, cd, chmod, chromium, chsh, convert, exa

### PR DESCRIPTION
**Pull request by FrankBG67**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/common/alias.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`alias`
+
+- Erstellt einen Alias:
+
+`alias {{wort}}="{{befehl}}"`
+
+- Zeigt den mit einem bestimmten Alias verknüpften Befehl an:
+
+`alias {{word}}`
```
Comment (zdroid) [inline pages.de/common/alias.md:16]:
```suggestion
`alias {{wort}}`
```

File: pages.de/common/alias.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Entferne einen Alias :
+
+`unalias {{word}}`
```
Comment (zdroid) [inline pages.de/common/alias.md:20]:
```suggestion
`unalias {{wort}}`
```

File: pages.de/common/borg.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Sichern Sie ein Verzeichnis in das Repository und erstellen Sie ein Archiv mit dem Namen "Montag":
+
+`borg create --progress {{/pfad/zum/repo_verzeichnis}}::{{Montag}} {{/pfad/zum/quell_verzeichnis}}`
+
+-  Alle Archive in einem Repository auflisten
+
+`borg list {{/pfad/zum/repo_verzeichnis}}`
+
+- Extrahieren eines bestimmten Verzeichnisses aus dem "Monday"-Archiv in einem entfernten Repository, unter Ausschluss aller *.ext-Dateien:
```
Comment (zdroid) [inline pages.de/common/borg.md:19]:
```suggestion
- Extrahieren eines bestimmten Verzeichnisses aus dem Archiv mit dem Namen "Montag" in einem entfernten Repository, unter Ausschluss aller *.ext-Dateien:
```
oder `dem Archiv benannt "Montag"`?

File: pages.de/common/chmod.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+- Gib anderen ([o]thers) (nicht in der Gruppe des Dateieigentümers) die gleichen Rechte wie der [g]ruppe:
+
+`chmod o=g {{file}}`
+
+- Entferne alle Rechtr der Anderen ([o]thers):
+
+`chmod o= {{file}}`
+
+- Ändere Genehmigungen rekursiv, indem Sie  [g]ruppe and Anderen ([o]thers) die Erlaubniss zum schreiben ([w]rite) geben:
```
Comment (zdroid) [inline pages.de/common/chmod.md:29]:
```suggestion
- Ändere Genehmigungen rekursiv, indem Sie [g]ruppe and Anderen ([o]thers) die Erlaubniss zum schreiben ([w]rite) geben:
```

File: pages.de/common/exa.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Listen Sie die Dateien mit den größten oben auf:
+
+`exa --reverse --sort={{size}}`
+
+- Zeigt Dateien in einer Baumstruktur an, der drei Ebenen tief ist:
+
+`exa --long --tree --level={{3}}`
+
+- Dateien nach Änderungsdatum sortiert auflisten (älteste zuerst)
```
Comment (zdroid) [inline pages.de/common/exa.md:26]:
```suggestion
- Dateien nach Änderungsdatum sortiert auflisten (älteste zuerst):
```

File: pages.de/common/alias.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Erstellt einen Alias:
+
+`alias {{wort}}="{{befehl}}"`
+
+- Zeigt den mit einem bestimmten Alias verknüpften Befehl an:
+
+`alias {{word}}`
+
+- Entferne einen Alias :
```
Comment (zdroid) [inline pages.de/common/alias.md:18]:
```suggestion
- Entferne einen Alias:
```

Comment (FrankBG67) [reply pages.de/common/alias.md:18]: Those were my first command descriptions.
I try to be more thorough.
I don't want the description to include a person who executes the command, but only a description of what the command does.
The first ones are flawed. I'm correct them.

Comment (zdroid) [reply pages.de/common/alias.md:18]: @FrankBG67 I think it's fine either way. I just wanted it to be consistent, so that one form is used in all examples.

**Comment by zdroid** [comment]
Great work! This is good to go.

**Pull request merged**